ui/public/i18n/language.json

```diff
@@ -378,7 +378,7 @@
     "renew_days": "Renew days",
     "request": "Request",
     "cockpit_reload_warning_title": "Notice.",
-    "cockpit_reload_warning_message": "The new certificate is applied to the Cockpit Server Manager 90 seconds after the last user disconnects.",
+    "cockpit_reload_warning_message": "The new certificate is applied to the Cockpit Server Manager 90 seconds after the last user disconnects. You cannot ask more of 10 certificate requests by 3 hours, the error of validation are included in the rate limit.",
```

Comment:
> You cannot ask more of 10 certificate requests by 3 hours, the error of validation are included in the rate limit.

This is not really true, it applies only to new account so in our case it's a limit or the validator. See https://letsencrypt.org/docs/rate-limits/

I'd remove the additional because it's a little out of context.

Reply:
done